crmsh/bootstrap.py

```diff
@@ -1303,6 +1303,17 @@ def init_sbd():
     SBD can also run in diskless mode if no device
     is configured.
     """
+    from .sbd import SBDManager
+
+    if _context.stage == "sbd" and utils.service_is_active("sbd.service"):
```

Comment:
What if sbd.service is somehow inactive but there's an existing sbd resource. Do we probably want to clean that up anyways?

Reply:
> What if sbd.service is somehow inactive but there's an existing sbd resource. Do we probably want to clean that up anyways?

When sbd.service is inactive(while cluster is running?), it seems impossible due to the service dependency only if sbd crash? Even it's possible, crmsh can not clean it up automatically, if user not do that manually.
BTW, Is harmful to keep this sbd resource running, when sbd.service is inactive?

Reply:
Thanks @gao-yan , changed